docroot/modules/humsci/hs_admin/bookmark-help/css/bookmark-help.css

```diff
@@ -26,6 +26,12 @@
   visibility: hidden;
 }
 
+/* Something happened with the styles of this input in gin so this fix it */
```

Comment:
Please make comment more descriptive. Also, this breaks the layout in mobile:

<img width="329" height="177" alt="Screenshot 2025-11-18 at 10 04 19 AM" src="https://github.com/user-attachments/assets/8ffb5e16-b7a7-4bba-9592-e6a6c67f5195" />

Please try something that doesn't use a hardcoded value. I tested something like this and seems to work better (the only problem is that there's some space left on the right, but it's not that bad):

```
min-width: unset !important;
width: calc(100% - 22px); // Leave space for the copy button.
```